.github/workflows/build.yml

```diff
@@ -212,7 +213,7 @@ jobs:
       - name: Set up GraalVM 17
         uses: graalvm/setup-graalvm@v1
         with:
-          version: 'latest'
+          version: 'mandrel-latest'
```

Comment:
OK, I will add it. So the following is OK?
```yaml
    - name: Set up Mandrel GraalVM JDK 17
        uses: graalvm/setup-graalvm@v1
        with:
          distribution: mandrel
          version: 'latest'
          java-version: 17
          ...
```

Reply:
LGTM :slightly_smiling_face:

Reply:
Done

Reply:
It is not working :(

```
Error: Unable to locate executable file: gu. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.
```

Mandrel distribution does not have `native-image`? We have to revert the this change.

Reply:
I have a very similar setup here: 
https://github.com/turing85/quarkus-qr/blob/2ecbe1b4461b503f2e93b30ada7f4970eaa9a4b7/.github/workflows/build.yml#L162-L170

Let me dig a little bit deeper. This should work.

Reply:
@zhfeng I fixated the version of mandrel in my pipelines (see e.g. [here (`github.com`)](https://github.com/turing85/quarkus-qr/actions/runs/7041616103/job/19164496120#step:3:6)). Can you try setting the `version` explicitly to `mandrel-23.0.1.2-Final`?

Reply:
Looking at this... in my pipelines, I do not set the `components`. This might also be a (the) cause of that issue. Can you try un-setting this variable?

Reply:
OK, I will check it.